schema.graphql

```diff
@@ -276,6 +308,30 @@ type PluginExtension {
   translatedDescription(lang: String): String! @goField(forceResolver: true)
 }
 
+# Widget Align System
+
+type WidgetAlignSystem {
+  inner: WidgetZone
+  outer: WidgetZone
+}
+
+type WidgetZone {
+  left: WidgetSection
+  center: WidgetSection
+  right: WidgetSection
+}
+
+type WidgetSection {
+  top: WidgetArea
+  middle: WidgetArea
+  bottom: WidgetArea
+}
+
+type WidgetArea {
+  widgetIds: [ID!]!
+  align: String
+}
+
```

Comment:
Move them under ScenePlugin and remove `# Widget Align System` because for now comments are for only root entities.